geopandas/base.py

```diff
@@ -289,13 +316,13 @@ def geom_almost_equals(self, other, decimal=6):
             Decimal place presion used when testing for approximate equality.
         """
         # TODO: pass precision argument
```

Comment:
I think I'm the one who missed this in a previous review, but that TODO can be removed now.